algorithms/vi-map-helpers/src/vi-map-landmark-quality-evaluation.cc

```diff
@@ -25,12 +43,28 @@ void evaluateLandmarkQuality(
     VLOG(1) << "Evaluating quality of landmarks of " << num_landmarks
             << " landmarks of mission " << mission_id.hexString();
 
+    const size_t num_threads = common::getNumHardwareThreads();
+    vi_map::TrackKeypointMapList outlier_track_ids_with_observations_per_thread(
+        num_threads);
+    const bool kAlwaysParallelize = false;
```

Comment:
You could still leave the old bit with evaluating quality to be parallel here and simply execute you function in a non parallel fashion afterwards in a separate loop? Or maybe even before everything splitting would be the first step and after that only would you do the quality evaluation because like that it would be fair and the new landmarks you create would also be evaluated. Like this at the moment they are skipped

Reply:
done that